src/router/index.js

```diff
@@ -24,7 +24,7 @@ const routes = [
     path: '/stats',
     redirect: () => {
       let now = new Date()
-      let month = `${now.getFullYear()}-${now.getMonth() + 1}`
+      let month = `${now.getFullYear()}-${String(now.getMonth() + 1).padStart(2, "0")}`
```

Comment:
We might want to consider using something like luxon to render things like this.

Reply:
Good point. I think for just one or two lines of code, a new dependency is a bit of an overkill, but we should definitely keep that in mind, in case we start getting more date handling code ...